src/main.cpp

```diff
@@ -2,12 +2,30 @@
 #include <syslog.h>
 #include "configInterface.h"
 
+bool dual_tor_sock = false;
+
+static void usage()
+{
+    printf("Usage: ./dhcp6relay {-d}\n");
+    printf("\t-u: enable dual tor option\n");
```

Comment:
The previous line has `-d`, but this line has `-u`.

Reply:
Thanks, updated